Changes

```diff
@@ -371,6 +371,10 @@ Working version
 - #13818: better delimited hints in error message
   (Florian Angeletti, review by Gabriel Scherer)
 
+- #13911: Added an `Lost alias` error message for deep destructive substitutions
```

Comment:
This change might break a few codes due to the new error message. Thus you should use a `*`